### PR TITLE
Added task description and use cases for zero-shot-classification

### PR DESCRIPTION
**Pull request by 02shanks**
Had to close previous PR huggingface/hub-docs#374 due to some issues. So created this new PR.

### CODE REVIEW

File: tasks/src/Zero-shot-classification/data.ts
Comment (merveenoyan) [inline tasks/src/Zero-shot-classification/data.ts:22]:
```suggestion
			id:          "",
```

File: tasks/src/Zero-shot-classification/about.md
Comment (merveenoyan) [inline tasks/src/Zero-shot-classification/about.md:5]: I feel like this is a very good definition for summary part instead 😅 WDYT?

File: tasks/src/Zero-shot-classification/data.ts
Comment (merveenoyan) [inline tasks/src/Zero-shot-classification/data.ts:25]: Can you simplify the language here a little for a broader audience 🙂

**Review by merveenoyan — COMMENTED** [review]
Thanks a lot for working on this! ❤️🙌🏻
Left a very general feedback, can you take a look? 🙂

File: tasks/src/Zero-shot-classification/about.md
Comment (02shanks) [reply tasks/src/Zero-shot-classification/about.md:5]: ya, that would be great. I will do the necessary changes

File: tasks/src/Zero-shot-classification/data.ts
Comment (02shanks) [reply tasks/src/Zero-shot-classification/data.ts:25]: I will try to simplify but will be adding zero-shot-text-classification work here?

Comment (02shanks) [reply tasks/src/Zero-shot-classification/data.ts:25]: @merveenoyan can you clear the doubts I mentioned above !!

**Comment by 02shanks** [comment]
@merveenoyan can you please review the content I changed according to the suggestion you provided me!!

File: tasks/src/Zero-shot-classification/about.md
Comment (merveenoyan) [reply tasks/src/Zero-shot-classification/about.md:5]: yes it's actually text classification https://huggingface.co/models?pipeline_tag=zero-shot-classification&sort=downloads

**Review by merveenoyan — APPROVED** [review]
I think we're all set after this one, I'm happy to merge if you could replace task summary.

**Comment by 02shanks** [comment]
@merveenoyan I made changes, can you review it!!

**Comment by 02shanks** [comment]
@merveenoyan can also review #387 for zero-shot-image-classification and suggest the changes!!

File: tasks/src/Zero-shot-classification/data.ts
Comment (merveenoyan) [inline tasks/src/Zero-shot-classification/data.ts:1]: Your data.ts template is missing two sections: demo & metrics. Can you add them back (can be empty) because the file has a template that needs to be followed.

**Review by merveenoyan — CHANGES_REQUESTED** [review]
Can you name the folder `zero-shot-classification` and add the missing sections back? The content looks good to me overall!

**Comment by 02shanks** [comment]
@merveenoyan I made changes, can you review it!!

**Comment by merveenoyan** [comment]
@02shanks I still see that the folder name is "Zero-shot-classification", can you rename as "zero-shot-classification"?

**Comment by 02shanks** [comment]
@merveenoyan I made changes, can you review it!!

**Comment by merveenoyan** [comment]
@02shanks can you copy and paste placeholder page for `about.md` and make your edits inside? makes it easier for others to contribute. 🙂

**Comment by merveenoyan** [comment]
Partially fixes #361

**Comment by 02shanks** [comment]
@merveenoyan I made changes, can you review it!!

**Comment by 02shanks** [comment]
@merveenoyan how can I resolve this conflict as its my first time encountering such thing?

**Comment by merveenoyan** [comment]
@02shanks So essentially what happens is that you were trying to write your changes to this repository's main branch. However, since you opened the PR, some changes occurred to that branch. Because of this, there's a conflict. Git says that "for fifth line of the code you wanted to write this, but this line is now changed, so it's better if you adapt your changes on top of these changes". 

Because you're working on a fork of original repository, first you need to have the changes (for main branch) To do so, simply sync your fork first: 
Ekran Resmi 2022-10-23 06 52 05

Then on your local repository, do a `git pull` while you're on main branch of your fork. Then, merge changes from main branch to the branches that your PRs are based on, so that they're updated too! This will bring the changes on your local and raise a merge conflict that you can solve through your IDE. Once you resolve the conflicts, simply add, commit and push your changes.
I hope this helps. I mainly wanted to write down instead of doing this myself as there might be more merge conflicts ahead (they occur a lot 🙂) I left a link below. 
https://www.simplilearn.com/tutorials/git-tutorial/merge-conflicts-in-git

File: tasks/src/zero-shot-classification/data.ts
Comment (merveenoyan) [inline tasks/src/zero-shot-classification/data.ts:59]:
```suggestion
	summary:      "Zero-shot text classification is the task of classifying classes previously unseen to the model during the training. ",
```

File: tasks/src/zero-shot-classification/about.md
Comment (merveenoyan) [inline tasks/src/zero-shot-classification/about.md:3]:
```suggestion
### Sentiment Analysis
```

File: tasks/src/zero-shot-classification/about.md
Comment (merveenoyan) [inline tasks/src/zero-shot-classification/about.md:5]:
```suggestion
Zero-shot sentiment analysis is the task of analyzing the sentiments of people in the text, without any training data for those sentiments. This can be done by using a model that was previously trained on a large dataset of images with labels for different sentiments and then using that model to predict the sentiments of people in new images.
```
Why images? This task is for text, no? I also don't see how it's zero shot given the model has already seen the sentiments previously (as you've written later on).

**Review by merveenoyan — COMMENTED** [review]
Thanks a lot for the effort you've put in this! Left two comments. 🙂

**Comment by 02shanks** [comment]
@merveenoyan I changed the description. Is it right now?

**Review by merveenoyan — APPROVED** [review]
We can merge 🙂

**Comment by 02shanks** [comment]
@merveenoyan thanks for guiding me patiently!!

**Comment by 02shanks** [comment]
@merveenoyan Is there any problem in this PR as it is not merged yet!!

**Comment by merveenoyan** [comment]
@02shanks I was waiting for you to commit my suggestion but I committed it, will merge now 🙂

**Pull request merged**
